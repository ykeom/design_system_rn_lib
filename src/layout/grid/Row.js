import React, { Component } from 'react';
import { View, StyleSheet, Text } from 'react-native';

const styles = StyleSheet.create({
  root: {},
});

class Row extends Component {
  render() {
    return (
      <View style={styles.root}>
        <Text>layout</Text>
      </View>
    );
  }
}

export default Row;
