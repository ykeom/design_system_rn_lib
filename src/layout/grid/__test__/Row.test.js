import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Row from '../Row';

describe('Row', () => {
  it('should render', () => {
    const component = shallow(<Row />);

    expect(component.length).toBe(1);
    expect(toJson(component)).toMatchSnapshot();
  });
});
