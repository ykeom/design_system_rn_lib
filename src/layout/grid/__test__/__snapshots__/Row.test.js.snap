// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row should render 1`] = `
<View
  style={Object {}}
>
  <Text>
    layout
  </Text>
</View>
`;
